### ISSUE TITLE
Unable to install LSPs on Windows

### ISSUE THREAD

**Issue by TiLopes** [williamboman/mason.nvim#1251]
### I've searched open issues for similar requests

Yes

### I've recently downloaded the latest plugin version of mason.nvim

- [X] Yes

### Problem description

Whenever I try to install an LSP I get a permission error saying that the operation is not permitted. It manages to download the file properly (I think).

### Expected behavior

Install an LSP and no errors occur.

### Affected packages

I believe all LSPs

### Mason output

```Text
Downloading file "https://github.com/luals/lua-language-server/releases/download/3.6.18/lua-language-server-3.6.18-win32-x64.zip"...
EPERM: operation not permitted: C:\Users\Tiago\AppData\Local\nvim-data\mason\packages\lua-language-server\mason-schemas\lsp.json -> C:\Users\Tiago\AppData\Local\nvim-data\mason\share\mason-schemas\lsp\lua-language-server.json
```

### Installation log

```Text
[INFO  22-Apr-23 23:16:23] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:166: Executing installer for Package(name=lua-language-server) {}
[ERROR 22-Apr-23 23:16:31] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:200: Installation failed for Package(name=lua-language-server) error="EPERM: operation not permitted: C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\packages\\lua-language-server\\mason-schemas\\lsp.json -> C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\share\\mason-schemas\\lsp\\lua-language-server.json"
[INFO  22-Apr-23 23:36:35] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:166: Executing installer for Package(name=lua-language-server) {}
[ERROR 22-Apr-23 23:36:42] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:200: Installation failed for Package(name=lua-language-server) error="EPERM: operation not permitted: C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\packages\\lua-language-server\\mason-schemas\\lsp.json -> C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\share\\mason-schemas\\lsp\\lua-language-server.json"
[INFO  22-Apr-23 23:37:10] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:166: Executing installer for Package(name=lua-language-server) {}
[INFO  22-Apr-23 23:37:15] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:193: Installation succeeded for Package(name=lua-language-server)
[INFO  22-Apr-23 23:37:24] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:166: Executing installer for Package(name=rust-analyzer) {}
[INFO  22-Apr-23 23:37:28] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:193: Installation succeeded for Package(name=rust-analyzer)
[INFO  22-Apr-23 23:39:14] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:166: Executing installer for Package(name=rust-analyzer) {}
[ERROR 22-Apr-23 23:39:16] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:200: Installation failed for Package(name=rust-analyzer) error="EPERM: operation not permitted: C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\packages\\rust-analyzer\\mason-schemas\\lsp.json -> C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\share\\mason-schemas\\lsp\\rust-analyzer.json"
[INFO  23-Apr-23 10:47:37] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:166: Executing installer for Package(name=rust-analyzer) {}
[ERROR 23-Apr-23 10:47:40] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:200: Installation failed for Package(name=rust-analyzer) error="EPERM: operation not permitted: C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\packages\\rust-analyzer\\mason-schemas\\lsp.json -> C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\share\\mason-schemas\\lsp\\rust-analyzer.json"
[INFO  23-Apr-23 10:50:53] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:166: Executing installer for Package(name=rust-analyzer) {}
[ERROR 23-Apr-23 10:50:56] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:200: Installation failed for Package(name=rust-analyzer) error="EPERM: operation not permitted: C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\packages\\rust-analyzer\\mason-schemas\\lsp.json -> C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\share\\mason-schemas\\lsp\\rust-analyzer.json"
[INFO  23-Apr-23 10:53:13] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:166: Executing installer for Package(name=lua-language-server) {}
[DEBUG 23-Apr-23 10:53:13] ...ta/Local/nvim-data/lazy/mason.nvim/lua/mason-core/fs.lua:71: fs: mkdirp C:\Users\Tiago\AppData\Local\nvim-data\mason\.packages\lua-language-server
[DEBUG 23-Apr-23 10:53:13] ...zy/mason.nvim/lua/mason-core/installer/registry/init.lua:145: Compiling installer. lua-language-server {}
[DEBUG 23-Apr-23 10:53:13] ...zy/mason.nvim/lua/mason-core/installer/registry/init.lua:107: Parsing spec lua-language-server {}
[DEBUG 23-Apr-23 10:53:13] ...azy/mason.nvim/lua/mason-core/installer/managers/std.lua:95: std: downloading file "https://github.com/luals/lua-language-server/releases/download/3.6.18/lua-language-server-3.6.18-win32-x64.zip"
[DEBUG 23-Apr-23 10:53:13] ...Local/nvim-data/lazy/mason.nvim/lua/mason-core/fetch.lua:34: Fetching URL "https://github.com/luals/lua-language-server/releases/download/3.6.18/lua-language-server-3.6.18-win32-x64.zip"
[DEBUG 23-Apr-23 10:53:13] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:116: Spawning cmd="curl", spawn_opts={
  args = { "-H", "User-Agent: mason.nvim (+https://github.com/williamboman/mason.nvim)", "-fsSL", "-X", "GET", "-o", "C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\.packages\\lua-language-server\\.\\lua-language-server-3.6.18-win32-x64.zip", "https://github.com/luals/lua-language-server/releases/download/3.6.18/lua-language-server-3.6.18-win32-x64.zip" }
}
[DEBUG 23-Apr-23 10:53:13] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:162: Spawned with pid 8624
[DEBUG 23-Apr-23 10:53:13] ...Local/nvim-data/lazy/mason.nvim/lua/mason-core/fetch.lua:34: Fetching URL "https://api.mason-registry.dev/api/github/mason-org/mason-registry/releases/latest"
[DEBUG 23-Apr-23 10:53:13] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:116: Spawning cmd="curl", spawn_opts={
  args = { "-H", "Accept: application/vnd.mason-registry.v1+json; q=1.0, application/json; q=0.8", "-H", "User-Agent: mason.nvim (+https://github.com/williamboman/mason.nvim)", "-fsSL", "-X", "GET", "https://api.mason-registry.dev/api/github/mason-org/mason-registry/releases/latest" }
}
[DEBUG 23-Apr-23 10:53:13] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:162: Spawned with pid 5100
[DEBUG 23-Apr-23 10:53:13] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:148: Job pid=5100 exited with exit_code=0, signal=0
[DEBUG 23-Apr-23 10:53:13] ...Local/nvim-data/lazy/mason.nvim/lua/mason-core/fetch.lua:34: Fetching URL "https://github.com/mason-org/mason-registry/releases/download/2023-04-23-irate-factor/registry.json.zip"
[DEBUG 23-Apr-23 10:53:13] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:116: Spawning cmd="curl", spawn_opts={
  args = { "-H", "User-Agent: mason.nvim (+https://github.com/williamboman/mason.nvim)", "-fsSL", "-X", "GET", "-o", "C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\registries\\github\\mason-org\\mason-registry\\registry.json.zip", "https://github.com/mason-org/mason-registry/releases/download/2023-04-23-irate-factor/registry.json.zip" }
}
[DEBUG 23-Apr-23 10:53:13] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:162: Spawned with pid 12044
[DEBUG 23-Apr-23 10:53:14] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:148: Job pid=12044 exited with exit_code=0, signal=0
[DEBUG 23-Apr-23 10:53:14] ...ta/Local/nvim-data/lazy/mason.nvim/lua/mason-core/fs.lua:59: fs: unlink C:\Users\Tiago\AppData\Local\nvim-data\mason\registries\github\mason-org\mason-registry\registry.json.zip
[DEBUG 23-Apr-23 10:53:14] ...Local/nvim-data/lazy/mason.nvim/lua/mason-core/fetch.lua:34: Fetching URL "https://github.com/mason-org/mason-registry/releases/download/2023-04-23-irate-factor/checksums.txt"
[DEBUG 23-Apr-23 10:53:14] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:116: Spawning cmd="curl", spawn_opts={
  args = { "-H", "User-Agent: mason.nvim (+https://github.com/williamboman/mason.nvim)", "-fsSL", "-X", "GET", "https://github.com/mason-org/mason-registry/releases/download/2023-04-23-irate-factor/checksums.txt" }
}
[DEBUG 23-Apr-23 10:53:14] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:162: Spawned with pid 10000
[DEBUG 23-Apr-23 10:53:14] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:148: Job pid=8624 exited with exit_code=0, signal=0
[DEBUG 23-Apr-23 10:53:14] ...azy/mason.nvim/lua/mason-core/installer/managers/std.lua:232: std: unpack "lua-language-server-3.6.18-win32-x64.zip"
[DEBUG 23-Apr-23 10:53:14] ...azy/mason.nvim/lua/mason-core/installer/managers/std.lua:122: std: unzip "lua-language-server-3.6.18-win32-x64.zip"
[DEBUG 23-Apr-23 10:53:14] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:116: Spawning cmd="pwsh", spawn_opts={
  args = { "-NoProfile", "-NonInteractive", "-Command", " $ErrorActionPreference = \"Stop\";  $ProgressPreference = 'SilentlyContinue';  [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; Microsoft.PowerShell.Archive\\Expand-Archive -Path \"lua-language-server-3.6.18-win32-x64.zip\" -DestinationPath ." },
  cwd = "C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\.packages\\lua-language-server\\.",
  env = { "GOPATH=C:\\Users\\Tiago\\go", "VIM=<redacted>", "WEZTERM_EXECUTABLE=<redacted>", "CHOCOLATEYTOOLSLOCATION=<redacted>", "HOMEPATH=<redacted>", "SYSTEMDRIVE=<redacted>", "PNPM_HOME=<redacted>", "PROGRAMDATA=<redacted>", "LOGONSERVER=<redacted>", "FNM_VERSION_FILE_STRATEGY=<redacted>", "FNM_ARCH=<redacted>", "JAVA_HOME=<redacted>", "TERM_PROGRAM=<redacted>", "COLORTERM=<redacted>", "USERNAME=<redacted>", "TEMP=<redacted>", "FNM_MULTISHELL_PATH=<redacted>", "HOME=<redacted>", "POSH_THEMES_PATH=<redacted>", "FNM_LOGLEVEL=<redacted>", "HOMEDRIVE=<redacted>", "COMSPEC=<redacted>", "=<redacted>", "STARSHIP_SHELL=<redacted>", "FPS_BROWSER_APP_PROFILE_STRING=<redacted>", "USERDOMAIN_ROAMINGPROFILE=<redacted>", "WEZTERM_EXECUTABLE_DIR=<redacted>", "COMMONPROGRAMW6432=<redacted>", "MYVIMRC=<redacted>", "LOCALAPPDATA=<redacted>", "COMPUTERNAME=<redacted>", "WEZTERM_CONFIG_DIR=<redacted>", "USERDOMAIN=<redacted>", "PROCESSOR_REVISION=<redacted>", "TMP=<redacted>", "CHOCOLATEYLASTPATHUPDATE=<redacted>", "FPS_BROWSER_USER_PROFILE_STRING=<redacted>", "PROCESSOR_LEVEL=<redacted>", "POWERSHELL_DISTRIBUTION_CHANNEL=<redacted>", "SYSTEMROOT=<redacted>", "TERM_PROGRAM_VERSION=<redacted>", "FNM_NODE_DIST_MIRROR=<redacted>", "PROCESSOR_ARCHITECTURE=<redacted>", "WEZTERM_UNIX_SOCKET=<redacted>", "PROCESSOR_IDENTIFIER=<redacted>", "DRIVERDATA=<redacted>", "FNM_DIR=<redacted>", "PROGRAMW6432=<redacted>", "CHOCOLATEYINSTALL=<redacted>", "WINDIR=<redacted>", "WSLENV=<redacted>", "PATHEXT=<redacted>", "OS=<redacted>", "WEZTERM_PANE=<redacted>", "NUMBER_OF_PROCESSORS=<redacted>", "PATH=C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\bin;C:\\Users\\Tiago\\AppData\\Local\\fnm_multishells\\4820_1682243072856;C:\\Program Files\\PowerShell\\7;C:\\Users\\Tiago\\AppData\\Local\\Programs\\Python\\Python311\\Scripts\\;C:\\Users\\Tiago\\AppData\\Local\\Programs\\Python\\Python311\\;C:\\Program Files\\Zulu\\zulu-19\\bin\\;C:\\Program Files\\ImageMagick-7.1.0-Q16-HDRI;C:\\Program Files (x86)\\VMware\\VMware Player\\bin\\;C:\\Program Files\\Eclipse Adoptium\\jdk-8.0.345.1-hotspot\\bin;C:\\Program Files\\Eclipse Adoptium\\jdk-17.0.4.101-hotspot\\bin;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\iCLS\\;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\iCLS\\;C:\\Program Files (x86)\\Razer Chroma SDK\\bin;C:\\Program Files\\Razer Chroma SDK\\bin;C:\\Program Files (x86)\\Razer\\ChromaBroadcast\\bin;C:\\Program Files\\Razer\\ChromaBroadcast\\bin;C:\\Program Files\\Eclipse Foundation\\jdk-16.0.2.7-hotspot\\bin;C:\\Windows\\system32;C:\\Windows;C:\\Windows\\System32\\Wbem;C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Windows\\System32\\OpenSSH\\;C:\\Program Files\\dotnet\\;C:\\Program Files (x86)\\NVIDIA Corporation\\PhysX\\Common;C:\\ProgramData\\chocolatey\\bin;C:\\Windows\\system32\\config\\systemprofile\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Program Files\\NVIDIA Corporation\\NVIDIA NvDLISR;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\DAL;C:\\Program Files (x86)\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\Program Files\\Intel\\Intel(R) Management Engine Components\\IPT;C:\\Program Files (x86)\\Windows Kits\\10\\Windows Performance Toolkit\\;C:\\Program Files\\CMake\\bin;C:\\Program Files\\Go\\bin;C:\\Users\\Tiago\\AppData\\Roaming\\Apps\\VSCodium\\bin;C:\\Program Files\\VSCodium\\bin;C:\\Program Files\\starship\\bin\\;C:\\Program Files\\Git\\cmd;C:\\Users\\Tiago\\AppData\\Roaming\\Apps\\MrCode\\bin;C:\\Program Files\\Docker\\Docker\\resources\\bin;C:\\Program Files\\PowerShell\\7\\;C:\\Program Files\\nodejs\\;C:\\Users\\Tiago\\.cargo\\bin;C:\\Users\\Tiago\\AppData\\Local\\pnpm;C:\\Users\\Tiago\\AppData\\Local\\pnpm;C:\\Users\\Tiago\\AppData\\Local\\pnpm;C:\\Users\\Tiago\\AppData\\Local\\Programs\\Python\\Launcher\\;C:\\Users\\Tiago\\scoop\\shims;C:\\tools\\neovim\\nvim-win64\\bin;C:\\Users\\Tiago\\AppData\\Local\\Microsoft\\WindowsApps;C:\\Program Files\\Eclipse Adoptium\\jdk-17.0.1.12-hotspot\\bin;C:\\Program Files\\Eclipse Foundation\\jdk-16.0.2.7-hotspot\\bin;C:\\Program Files\\Eclipse Adoptium\\jdk-8.0.322.6-hotspot\\bin;C:\\msys64\\mingw64\\bin;C:\\msys64\\usr\\bin;C:\\Users\\Tiago\\AppData\\Local\\Programs\\oh-my-posh\\bin;C:\\Users\\Tiago\\AppData\\Local\\Programs\\oh-my-posh\\themes;C:\\Users\\Tiago\\.dotnet\\tools;C:\\Users\\Tiago\\AppData\\Local\\Programs\\Microsoft VS Code\\bin;C:\\Users\\Tiago\\adb-fastboot\\platform-tools;C:\\tools\\dart-sdk\\bin;C:\\Users\\Tiago\\AppData\\Roaming\\Pub\\Cache\\bin;C:\\ProgramData\\chocolatey\\lib\\mingw\\tools\\install\\mingw64\\bin;C:\\xampp\\php;C:\\Users\\Tiago\\AppData\\Local\\ComposerSetup\\bin;C:\\Users\\Tiago\\AppData\\Roaming\\Composer\\vendor\\bin;C:\\Users\\Tiago\\Documents\\PowerShell\\Scripts;C:\\Users\\Tiago\\AppData\\Local\\spicetify;C:\\tools\\lua;C:\\Users\\Tiago\\go\\bin;C:\\Program Files\\Oracle\\VirtualBox;C:\\Users\\Tiago\\AppData\\Roaming\\TinyTeX\\bin\\win32;C:\\Program Files\\7-Zip\\;C:\\Users\\Tiago\\go\\bin;C:\\tools;D:\\.bin;C:\\Users\\Tiago\\AppData\\Local\\JetBrains\\Toolbox\\scripts;C:\\Users\\Tiago\\AppData\\Local\\Programs\\VSCodium\\bin;C:\\Users\\Tiago\\AppData/Local\\bob\\nvim-bin;C:\\Users\\Tiago\\AppData/Local\\bob\\nvim-bin;C:\\Program Files\\RedHat\\Podman;C:\\Users\\Tiago\\AppData\\Roaming\\npm;C:\\Users\\Tiago\\scoop\\apps\\neovim\\current\\bin;C:\\Users\\Tiago\\scoop\\apps\\neovim\\current\\bin;C:\\Users\\Tiago\\AppData\\Local\\nvim-data/mason/bin", "PUBLIC=<redacted>", "MASON=<redacted>", "ONEDRIVE=<redacted>", "NVIM_LOG_FILE=<redacted>", "COMMONPROGRAMFILES=<redacted>", "COMMONPROGRAMFILES(X86)=<redacted>", "STARSHIP_SESSION_KEY=<redacted>", "PROGRAMFILES(X86)=<redacted>", "VIRTUAL_ENV_DISABLE_PROMPT=<redacted>", "CLINK_DIR=<redacted>", "VIMRUNTIME=<redacted>", "WEZTERM_CONFIG_FILE=<redacted>", "SESSIONNAME=<redacted>", "ALLUSERSPROFILE=<redacted>", "PROGRAMFILES=<redacted>", "TERM=<redacted>", "APPDATA=<redacted>", "USERPROFILE=<redacted>" }
}
[DEBUG 23-Apr-23 10:53:14] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:162: Spawned with pid 11676
[DEBUG 23-Apr-23 10:53:14] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:148: Job pid=10000 exited with exit_code=0, signal=0
[DEBUG 23-Apr-23 10:53:18] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:148: Job pid=11676 exited with exit_code=0, signal=0
[DEBUG 23-Apr-23 10:53:18] ...ta/Local/nvim-data/lazy/mason.nvim/lua/mason-core/fs.lua:59: fs: unlink C:\Users\Tiago\AppData\Local\nvim-data\mason\.packages\lua-language-server\.\lua-language-server-3.6.18-win32-x64.zip
[DEBUG 23-Apr-23 10:53:18] ...mason.nvim/lua/mason-core/installer/registry/schemas.lua:49: schemas: download Package(name=lua-language-server) {
  lsp = "vscode:https://raw.githubusercontent.com/LuaLS/vscode-lua/master/package.json"
}
[DEBUG 23-Apr-23 10:53:18] ...ta/Local/nvim-data/lazy/mason.nvim/lua/mason-core/fs.lua:65: fs: mkdir C:\Users\Tiago\AppData\Local\nvim-data\mason\.packages\lua-language-server\mason-schemas
[DEBUG 23-Apr-23 10:53:18] ...Local/nvim-data/lazy/mason.nvim/lua/mason-core/fetch.lua:34: Fetching URL "https://raw.githubusercontent.com/LuaLS/vscode-lua/master/package.json"
[DEBUG 23-Apr-23 10:53:18] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:116: Spawning cmd="curl", spawn_opts={
  args = { "-H", "User-Agent: mason.nvim (+https://github.com/williamboman/mason.nvim)", "-fsSL", "-X", "GET", "https://raw.githubusercontent.com/LuaLS/vscode-lua/master/package.json" }
}
[DEBUG 23-Apr-23 10:53:18] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:162: Spawned with pid 8952
[DEBUG 23-Apr-23 10:53:19] ...cal/nvim-data/lazy/mason.nvim/lua/mason-core/process.lua:148: Job pid=8952 exited with exit_code=0, signal=0
[DEBUG 23-Apr-23 10:53:19] ...zy/mason.nvim/lua/mason-core/installer/registry/link.lua:150: Registering bin links Package(name=lua-language-server) {
  ["lua-language-server"] = "{{source.asset.bin}}"
}
[DEBUG 23-Apr-23 10:53:19] ...zy/mason.nvim/lua/mason-core/installer/registry/link.lua:180: Expanded bin link "lua-language-server" -> "bin/lua-language-server.exe"
[DEBUG 23-Apr-23 10:53:19] ...ata/lazy/mason.nvim/lua/mason-core/installer/context.lua:212: Promoting cwd "C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\.packages\\lua-language-server" to "C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\packages\\lua-language-server"
[DEBUG 23-Apr-23 10:53:19] ...ta/Local/nvim-data/lazy/mason.nvim/lua/mason-core/fs.lua:84: fs: rename C:\Users\Tiago\AppData\Local\nvim-data\mason\.packages\lua-language-server C:\Users\Tiago\AppData\Local\nvim-data\mason\packages\lua-language-server
[DEBUG 23-Apr-23 10:53:19] ...data/lazy/mason.nvim/lua/mason-core/installer/linker.lua:129: Linking Package(name=lua-language-server)
[ERROR 23-Apr-23 10:53:19] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:200: Installation failed for Package(name=lua-language-server) error="EPERM: operation not permitted: C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\packages\\lua-language-server\\mason-schemas\\lsp.json -> C:\\Users\\Tiago\\AppData\\Local\\nvim-data\\mason\\share\\mason-schemas\\lsp\\lua-language-server.json"
[DEBUG 23-Apr-23 10:53:19] ...ta/Local/nvim-data/lazy/mason.nvim/lua/mason-core/fs.lua:47: fs: rmrf C:\Users\Tiago\AppData\Local\nvim-data\mason\packages\lua-language-server
[DEBUG 23-Apr-23 10:53:19] ...m-data/lazy/mason.nvim/lua/mason-core/installer/init.lua:39: Building receipt for Package(name=lua-language-server)
[DEBUG 23-Apr-23 10:53:19] ...data/lazy/mason.nvim/lua/mason-core/installer/linker.lua:42: Unlinking Package(name=lua-language-server)
[DEBUG 23-Apr-23 10:53:19] ...ta/Local/nvim-data/lazy/mason.nvim/lua/mason-core/fs.lua:59: fs: unlink C:\Users\Tiago\AppData\Local\nvim-data\mason\bin\lua-language-server.cmd
```

### Neovim version (>= 0.7)

NVIM v0.9.0
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3

### Operating system/version

Windows

### Healthcheck

```Text
mason: require("mason.health").check()

mason.nvim ~
- OK neovim version >= 0.7.0

mason.nvim [Registries] ~
- OK Registry `github.com/mason-org/mason-registry version: 2023-04-23-irate-factor` is installed.

mason.nvim [Core utils] ~
- WARNING unzip: not available
  - ADVICE:
    - spawn: unzip failed with exit code - and signal -. unzip is not executable
- OK wget: `GNU Wget 1.21.3 built on cygwin.`
- OK curl: `curl 8.0.1 (Windows) libcurl/8.0.1 Schannel WinIDN
`
- OK gzip: `gzip.exe" 1.3.12`
- OK tar: `bsdtar 3.5.2 - libarchive 3.5.2 zlib/1.2.5.f-ipp 
`
- OK pwsh: `7.3.4 Microsoft Windows 10.0.19045 Win32NT
`
- OK 7z: `7-Zip 22.01 (x64) : Copyright (c) 1999-2022 Igor Pavlov : 2022-07-15
`

mason.nvim [Languages] ~
- WARNING Ruby: not available
  - ADVICE:
    - spawn: ruby failed with exit code - and signal -. ruby is not executable
- WARNING RubyGem: not available
  - ADVICE:
    - spawn: gem.cmd failed with exit code - and signal -. gem.cmd is not executable
- OK Go: `go version go1.19.5 windows/amd64`
- OK PHP: `PHP 8.2.0 (cli) (built: Dec  6 2022 15:31:23) (ZTS Visual C++ 2019 x64)`
- OK cargo: `cargo 1.69.0 (6e9a83356 2023-04-12)`
- OK luarocks: `luarocks 3.9.1
`
- OK node: `v18.14.2
`
- WARNING julia: not available
  - ADVICE:
    - spawn: julia failed with exit code - and signal -. julia is not executable
- OK python: `Python 3.11.3
`
- OK java: `openjdk version "19.0.1" 2022-10-18
`
- OK javac: `javac 19.0.1
`
- WARNING pip: not available
  - ADVICE:
    - spawn: python failed with exit code 1 and signal 0. C:\Users\Tiago\AppData\Local\Programs\Python\Python311\python.exe: No module named pip

- OK JAVA_HOME: `openjdk version "17.0.4.1" 2022-08-12
`
- OK Composer: `Composer version 2.3.9 2022-07-05 16:52:11
`
- OK npm: `9.5.0`

mason.nvim [GitHub] ~
- OK GitHub API rate limit. Used: 4. Remaining: 56. Limit: 60. Reset: 23-Apr-23 11:23:30.
  Install and authenticate via gh-cli to increase rate limit.
```

**Comment by aiyolo** [comment]
I also encountered this problem

**Comment by williamboman** [comment]
Hey, should be fixed in #1253. Try pulling latest plugin version.
